main.py

```diff
@@ -1,10 +1,47 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+    for nickname, players_info in data.items():
```

Comment:
Ensure that the JSON structure matches the expected format in the code. If the JSON file structure changes, this loop might break.